packages/ui-components/src/components/info-label/info-label.tsx

```diff
@@ -0,0 +1,110 @@
+import { Component, Prop, h, Host, Element, State } from '@stencil/core';
+import { ResizeSensor } from 'css-element-queries';
+import clipboardHelper, { COPY_TOOLTIP } from '../../utils/clipboard.helper';
+
+@Component({
+	tag: 'kv-info-label',
+	styleUrls: {
+		night: 'info-label.night.scss',
+		light: 'info-label.light.scss'
+	},
+	shadow: true
+})
+export class KvInfoLabel {
+	private descriptionContainer: Element;
+	private tooltipConfig = COPY_TOOLTIP;
+
+	@State() enableShowMoreButton = false;
```

Comment:
Please add docs for the state.

Reply:
Probably `visibleShowMoreButton ` is a better name property than enable